setup.py

```diff
@@ -8,10 +8,8 @@
 extras_require = {
     "aws": ["aiobotocore>=0.10.2"],
     "azure": [
-        "azure-mgmt-compute>=18.0.0,<19",
-        "azure-mgmt-network>=16.0.0,<17",
-        "azure-cli-core>=2.15.1,<2.21.0",
-        "msrestazure",
+        "azure-mgmt-compute>=18.0.0",
```

Comment:
Could we keep the upper bound pins here, just move them on to a future major version? The azure packages don't seem afraid of large breaking changes and major version bumps.

Reply:
I don't actually know if these libraries use semver, so I don't know what "upper" would be here (the one change that actual did break things was the change to azure-cli-core in a "minor" release, if it is indeed following semver).

Either way, I'd probably prefer not to put an upper bound since if things *aren't* broken the user doesn't have an easy way to override the pin. If things do break, then the user can set an upper bound in their requirements while we (I) investigate (https://iscinumpy.dev/post/bound-version-constraints/ goes through this in detail).

Reply:
This makes sense, I've definitely been bitten by breaking changes in the Azure tools, and the major version seems to bump a lot so I assumed it was because of that. But if it isn't let's go without the pins for now.